### PR TITLE
(bug) Navbar style changes and glyphicons added

### PR DESCRIPTION
**Pull request by jefQuery**
Stylin', whilen, livin' it up in the city.
if user is logged in and display ‘Hi, user!’
My Rooms is not be rendered on the navbar unless the user is logged in
Filled in icon circle placeholders for each section/feature with glyphicons
Only displays the current room name when in a room, without displaying it on the main page
Logo moved to be further to the left and navigation more to the right
Logo appear as a dedicated home button, even in a small window

### CODE REVIEW

File: client/components/homepage/Features.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ const Features = () => (
       <div className="container">
         <div className="row text-center-mobile">
           <div className="col-md-6">
-            <div className="icon"><i className="pe-7s-diamond" /></div>
+            <div className="icon"><i className="glyphicon glyphicon-list-alt" /></div>
```
Comment (misteryeo) [inline client/components/homepage/Features.jsx]: Great job with adding the glyphicons - I'd like to take a look at how to center them within the circle though. Maybe @dshelters can also have a look at this while he restyles other aspect of the landing page.

File: client/styles.css [hunk 2]
```diff
@@ -483,6 +483,9 @@ INTRO
     background: #46426f;
   }
 }
+.userName {
+  color: #46426f
```
Comment (misteryeo) [inline client/styles.css]: I'm wondering if we can un-bold the 'Hi, {username}' and add a bit more spacing between it and the profile button. Looks a bit cramped at the moment. cc: @dshelters

**Comment by jefQuery** [comment]
[NavBar style bugs](https://trello.com/c/SsvqdzbN/127-navbar-style-bugs)

File: client/styles.css (hunk 2 above)
Comment (dshelters) [inline client/styles.css]: I'll take a look!

**Pull request merged**